### DISCUSSION TITLE
How Can I lower the sample rate?

### DISCUSSION THREAD

**Discussion by JianminWu** [apache/skywalking#10806] [Q&A] [answered]
I use sw 9.3.0
storage: MySQL
When I start sw MySQL load will become very high..
Can I lower the sample rate ? 
I see agent.conf agent.sample_n_per_3_secs ,but this config is a number value.
Can I use a percent value? such as 80% 90%?
Thanks!

**Comment by wu-sheng** [comment, marked as answer]
You could only use rate sampling on the OAP side. Agent side sampling reduces the payload of agent but easily causes inaccurate metrics. But the metrics would still accurate when sampling at the backend.

**Reply by JianminWu** [reply]
do you mean OAP application.yml  receiver-browse.default.sampleRate?

**Reply by wu-sheng** [reply]
No, it is about browser. Read trace sampling doc of backend rather than guessing.
